SemiTruck.h

```diff
@@ -1,2 +1,12 @@
 #pragma once
 
+#include "Vehicle.h"
+
+struct SemiTruck : Vehicle
+{
+    SemiTruck(const std::string& name);
+    ~SemiTruck();
+    
+    SemiTruck(const SemiTruck&) = default;
+    SemiTruck& operator=(const SemiTruck&) = default;
+};
```

Comment:
- [ ] add a member function to call when entering the highway.
- [ ] add a member function to call when getting pulled over.